index.ts

```diff
@@ -0,0 +1,142 @@
+export const auth = require('./lib/auth/auth');
```

Comment:
Is there a way around this massive index file? Like some sort of auto export or export * thing? I have forgotten to add new files to this at least 3 times...

Reply:
We can surely do something about that in the future, but for now, the idea is to keep the same structure for this PR. Later, we could do something like `export * as auth from 'auth-folder'` for every sub-folder. I think it's cleaner to do this for every future package.

Reply:
Ok, that is what I would expect. Sounds good.